In [ ]:
<a href="https://colab.research.google.com/github/samuel-apls/curso-python-gustavo-guanabra/blob/main/Python_Mundo_1_Desafios.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

**Desafio 001**

Crie um script Python que leia o nome de uma pessoa e mostra uma mensagem de boas-vindas.

In [ ]:
nome = input('Qual é seu nome? ')
print('Olá, {}! Prazer em te conhecer!'.format(nome))


**Desafio 002**

Crie um script Python que leia o dia, mês e o ano de nascimento de uma pessoa e mostra uma mensagem com a data formatada.

In [ ]:
dia = int(input('Digite o dia do seu nascimento: '))
mes = int(input('Digite o mês do seu nascimento: '))
ano = int(input('Digite o ano do seu nascimento: '))
print('Sua data de nascimento é: {}/{}/{}'.format(dia, mes, ano))


**Desafio 003**

Crie um programa que leia dois números e mostra a soma entre eles

In [ ]:
n1 = int(input('Digite um número: '))
n2 = int(input('Digite outro número: '))
print('A soma entre {} e {} é {}'.format(n1, n2, n1 + n2))


**Desafio 004**

Faça um programa que leia algo pelo teclado e mostra na tela seu tipo primitivo e todas as informações possíveis sobre ela

In [1]:
v = input('Digite algo: ')
print('Seu tipo primitivo é {}'.format(type(v)))
print('É numérico: {}'.format(v.isnumeric()))
print('É alfanúmerico: {}'.format(v.isalnum()))
print('É alfabético: {}'.format(v.isalpha()))
print('Está em maiúsculo: {}'.format(v.isupper()))


Digite algo: algo
Seu tipo primitivo é <class 'str'>
É numérico: False
É alfanúmerico: True
É alfabético: True
Está em maiúsculo: False


**Desafio 005**

Faça um programa que leia um número inteiro e mostra na tela seu sucessor e antecessor

In [4]:
numero = int(input('Digite um número: '))
print(f'Antecessor: {numero - 1}')
print(f'Sucessor: {numero + 1}')

Digite um número: 1
Antecessor: 0
Sucessor: 2


**Desafio 006**

Crie um algoritmo que leia um número e mostre o seu dobro, triplo e raíz quadrada

In [11]:
numero = float(input('Digite um número: '))
print(f'Dobro: {numero * 2:.0f}')
print(f'Triplo: {numero * 3:.0f}')
print(f'Raiz quadrada: {numero ** 2:.0f}')

Digite um número: 5
Dobro: 10
Triplo: 15
Raiz quadrada: 25


**Desafio 007**

Desenvolva um programa que leia as duas notas de um aluna, calcule mostre a sua média

In [12]:
nota1 = float(input('Primeira nota: '))
nota2 = float(input('Primeira nota: '))
media = (nota1 + nota2) / 2
print(f'Média: {media:.2f}')

Média: 3.75


**Desafio 008**

Escreva um programa que leia um valor em metros e o exiba convertido em centimetros e milímetros

**Desafio 009**

Fala um programa que leia um número inteiro qualquer e mostre na tela a sua tabuada

**Desafio 010**

Crie um programa que leia quanto dinheiro uma pessoa tem na carteira e mostre quantos dólares ela pode comprar (consideara US$ 1 = $$3,27)

**Desafio 011**

Faça um program que leia a largura e a altura de uma parede em metros, calcule a sua área e a quantidade de tinta becessária para pintá-la, sabendo que cada litro de tinta pinta uma área de 2m quadrados.

**Desafio 012**

Faça um algoritmo que leia o preço de um produto e mostre seu novo preço com 5% de desconto. 

**Desafio 013**

Faça um algoritmo que leia o salário de um funcionário e mostre seu novo salário, com 15% de aumento.